src/Maker/MakeUser.php

```diff
@@ -147,11 +147,10 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
 
         // B) Implement UserInterface
         $manipulator = new ClassSourceManipulator(
-            $this->fileManager->getFileContents($classPath),
-            true,
-            !$useAttributesForDoctrineMapping,
-            true,
-            $useAttributesForDoctrineMapping
+            sourceCode: $this->fileManager->getFileContents($classPath),
+            overwrite: true,
+            fluentMutators: true,
+            useAttributesForDoctrineMapping: $useAttributesForDoctrineMapping
```

Comment:
Is this still valid? Like... when would `$useAttributesForDoctrineMapping` be false... XML? annotations? If annotations, should we throw an exception? If XML... should we throw an exception?

Reply:
yes - well kinda, we still need to be able to generate user _models_ if the class isn't an entity. I've added a legacy && todo notation a few lines above to fix this in the pr im working on now.